### DISCUSSION TITLE
Is it compatible with nvidia video super resolution ?

### DISCUSSION THREAD

**Discussion by momala454** [VirtualDrivers/Virtual-Display-Driver#214] [Q&A] [answered]
How to use with the features nvidia video super resolution and rtx hdr to use with the browsers ?

**Comment by bud3699** [comment]
Since the display isn't available in nvidia control panel, I doubt you'll be able to, I've emailed nvidia support about integrating it but they haven't responded back

**Reply by momala454** [reply]
do you think it's possible to "hack" something to make the nvidia control panel show it ?

**Reply by bud3699** [reply]
> do you think it's possible to "hack" something to make the nvidia control panel show it ?

As of now, most likely not, I need to figure out how nvidia handles what gpus show, then even if i manage that, somehow figure out how to incorporate everything control panel can change.. and when theres no real display, that becomes quite a challenge

**Reply by momala454** [reply]
and spoofing the hardware id of a nvidia gpu ?

**Reply by bud3699** [reply]
its a possibilty but i highly doubt it. Even if it did show would most likely cause a lot of problems and introduce legal problems

**Reply by momala454** [reply]
so even if I have a RTX card, the feature only works on "monitors" plugged into that gpu, so it doesn't work on virtual monitors or monitors plugged in another gpu ?

**Reply by bud3699** [reply]
More specifically on later cards, in testing, the features in my games for feature levels stay active but i'm on a 20 series card. I'm assuming you're on a 30 series of 40 series. My guess is automatic power management on later gpus which cause problems

**Reply by momala454** [reply]
right now i'm on a 20 series, and it's unfortunate that even dlss is not working, that's very problematic ( https://github.com/VirtualDisplay/Virtual-Display-Driver/issues/114 )

edit: i suppose I need a virtual monitor dongle to be able to use dlss, nvidia video super resolution etc ?

**Reply by bud3699** [reply]
yeh, theres only so much we can do while staying on the legal side of things

**Reply by momala454** [reply]
i have just tested cyberpunk 2077, and dlss and ray tracing works. i have 2 monitors plugged into my rtx 2080, but they are disabled. And I only have the virtual monitor active.
Am I missing something ?

**Reply by bud3699** [reply, marked as answer]
Nope, just how some of the games check if its compatible, technically it is for all, just whether or not the game wants to show it if no displays are active on the connected gpu
